test/store.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
missing -e and pipefail

Reply:
This file is sourced so I think `-e` and `pipefail` aren't' appropriate here